### PR TITLE
feat(pci-object-storage): show object

### PR DESCRIPTION
**Pull request by sidlynx**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-object-storage-`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTCORE-2885
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-object-storage/public/translations/common/Messages_de_DE.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{
```
Comment (frenautvh) [inline packages/manager/apps/pci-object-storage/public/translations/common/Messages_de_DE.json:1]: you cam import common translations from pci-common using "pci-common" key, no need to copy them in the app

File: packages/manager/apps/pci-object-storage/src/api/hooks/useAvailability.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query';
+import { getFeatureAvailability } from '@/api/data/feature-availablity';
+
+export const useGetFeatureAvailability = (feature: string, app: string) => {
```
Comment (frenautvh) [inline packages/manager/apps/pci-object-storage/src/api/hooks/useAvailability.ts:4]: you can use useFeatureAvailability from manager-react-components

File: packages/manager/apps/pci-object-storage/src/api/hooks/useRegion.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { useQuery } from '@tanstack/react-query';
+import { getRegion } from '@/api/data/region';
+
+export const useGetRegion = (projectId: string, region: string) => {
```
Comment (frenautvh) [inline packages/manager/apps/pci-object-storage/src/api/hooks/useRegion.ts:4]: you can use useRegion from pci-common

**Comment by frenautvh** [comment]
could you please take a quick look on sonar issues to check what can quickly be fixed ? thanks

**Review by kqesar — APPROVED** [review]

**Pull request merged**